api/package.json

```diff
@@ -17,7 +17,8 @@
     "build-watch": "tsc --watch ",
     "serve-nodemon": "nodemon ./dist/index server",
     "serve": "run-p build-watch serve-nodemon",
-    "generate-openapi-schemas": "ts-node ./src/tools/open-api-creator/index.ts"
+    "generate-openapi-schemas": "ts-node ./src/tools/open-api-creator/index.ts",
```

Comment:
Please remove this from this branch and insert in the branch related to client library creation

Reply:
This script was already included, just marked as new line because of the added comma.